esp/platform/txsummary.cpp

```diff
@@ -81,8 +97,41 @@ bool CTxSummary::set(const char* key, const char* value)
     return true;
 }
 
+CumulativeTimer* CTxSummary::getTimer(const char* name)
```

Comment:
Our convention is normally to call the function queryTimer if it does not return a linked object.